### DISCUSSION TITLE
Rebuilding autokeras model from scratch

### DISCUSSION THREAD

**Discussion by plebik** [keras-team/autokeras#1492] [Q&A]
I would like to ask, is it possible to rebuild an autokeras model from scratch and getting the same or at least similar results. I know that I can see the structure of the model but it is hard for me to rebuild it. This is my summary

![keras](https://user-images.githubusercontent.com/49209280/105866586-6ec51200-5ff4-11eb-8b44-aff7437caf1f.png)

this is config of this model

![config1](https://user-images.githubusercontent.com/49209280/105873423-c1ee9300-5ffb-11eb-9f6c-ae3830223710.png)
![config2](https://user-images.githubusercontent.com/49209280/105873426-c2872980-5ffb-11eb-85a2-b758a3070af3.png)
![config3](https://user-images.githubusercontent.com/49209280/105873427-c2872980-5ffb-11eb-870e-7b583180215c.png)

and some other settings
![config4](https://user-images.githubusercontent.com/49209280/105874670-23633180-5ffd-11eb-9c91-430f0fce4d77.png)

**Comment by haifeng-jin** [comment]
If you want to get all the details of a model, you will have to read the source code of AutoKeras.
May I know why are you trying to rebuild it?

**Comment by stancsz** [comment]
as long as you don't overwrite your model
```
clf = ak.StructuredDataClassifier(overwrite=True, max_trials=3, overwrite=False)
```
doing so will load your automodel weights from the default path.
if you want to specify a specific path
```
clf = ak.StructuredDataClassifier(max_trials=3, overwrite=False, directory='mymodelgoeshere')
```
